client/interfaces.go

```diff
@@ -279,6 +282,28 @@ type ContainerSnapshotCopyArgs struct {
 	Live bool
 }
 
+// The ContainerConsoleArgs struct is used to pass additional options during a
+// container console session
+//
+// API extension: console
```

Comment:
Only shared/api/* uses the "API extension" comments for functions and data structures.
In client/* we just put hasExtension() checks in the various functions.

Reply:
done